doc/platform_api/CMIS_and_C-CMIS_support_for_ZR.md

```diff
@@ -0,0 +1,1332 @@
+  ```
+  admin@sonic:~$ show interfaces transceiver presence Ethernet0
+  Port         Low-power Mode
+  -----------  ----------------
+  Ethernet0  On
+  ```
+
+- Example (Display status of transceiver connected to Ethernet0):
+  ```
+  admin@sonic:~$ show interfaces transceiver status Ethernet0
+  Ethernet0:
+  Module State : Ready
+  Module Fault Cause : No Fault detected
+  Datapath Firmware Fault : False
+  Module Firmware Fault : False
+  Module State Changed : False
+  Datapath Host Lane 1: Activated
+  Datapath Host Lane 2: Activated
+  Datapath Host Lane 3: Activated
+  Datapath Host Lane 4: Activated
+  Datapath Host Lane 5: Activated
+  Datapath Host Lane 6: Activated
+  Datapath Host Lane 7: Activated
+  Datapath Host Lane 8: Activated
+  Tx Output Status : Valid
+  Rx Output Status : Valid  
+  Tx Fault : False
+  Tx Los : False
+  Tx Cdr Lol: False
+  Rx Los : False
+  Rx Cdr Lol : False
+  Configuration State Host Lane 1: Config Success
+  Configuration State Host Lane 2: Config Success
+  Configuration State Host Lane 3: Config Success
+  Configuration State Host Lane 4: Config Success
+  Configuration State Host Lane 5: Config Success
+  Configuration State Host Lane 6: Config Success
+  Configuration State Host Lane 7: Config Success
+  Configuration State Host Lane 8: Config Success
+  Datapath Init Pending Host Lane 1: DPInit not pending
+  Datapath Init Pending Host Lane 2: DPInit not pending
+  Datapath Init Pending Host Lane 3: DPInit not pending
+  Datapath Init Pending Host Lane 4: DPInit not pending
+  Datapath Init Pending Host Lane 5: DPInit not pending
+  Datapath Init Pending Host Lane 6: DPInit not pending
+  Datapath Init Pending Host Lane 7: DPInit not pending
+  Datapath Init Pending Host Lane 8: DPInit not pending
+  Tuning In Progress: Tuning Not In Progress
+  Wavelegnth Unlock Status : Wavelength Locked
+  Target Output Power Out Of Range : False
+  Fine Tuning Out Of Range : False
+  Tuning Not Accepted : False
+  Invalid Channel Number : False
+  Tuning Complete : True
+  Temperature High Alarm : False
+  Temperature High Warning : False
+  Temperature Low Alarm : False
+  Temperature Low Warning : False
+  Vcc High Alarm : False
+  Vcc High Warning : False
+  Vcc Low Alarm : False
+  Vcc Low Warning : False
+  Tx Power High Alarm : False
+  Tx Power High Warning : False
+  Tx Power Low Alarm : False
+  Tx Power Low Warning : False
+  Rx Power High Alarm : False
+  Rx Power High Warning : False
+  Rx Power Low Alarm : False
+  Rx Power Low Warning : False  
+  Tx Bias High Alarm : False
+  Tx Bias High Warning : False
+  Tx Bias Low Alarm : False
+  Tx Bias Low Warning : False  
+  Laser Temperature High Alarm : False
+  Laser Temperature High Warning : False
+  Laser Temperature Low Alarm : False
+  Laser Temperature Low Warning : False
+  ```
+- Example (Display pm status of transceiver connected to Ethernet0):
+  ```
+  admin@sonic:~$ show interfaces transceiver pm Ethernet0
```

Comment:
what is pm?

Reply:
Performance monitoring (PM) shows the statistics of a VDM value, including average, min and max values in a PM period. Please see Section 7.4.8 in C-CMIS document from OIF for more details: https://www.oiforum.com/wp-content/uploads/OIF-C-CMIS-01.1.pdf